src/main/java/com/github/vkorobkov/jfixtures/domain/Table.java

```diff
@@ -3,35 +3,58 @@
 import lombok.Getter;
 import lombok.val;
 
-import java.util.Collection;
-import java.util.Collections;
-import java.util.LinkedHashMap;
-import java.util.stream.Collectors;
+import java.util.*;
 import java.util.stream.Stream;
 
+import static java.util.Collections.*;
+import static java.util.Collections.emptyList;
+import static java.util.Collections.unmodifiableCollection;
+import static java.util.stream.Collectors.toList;
+import static java.util.stream.Collectors.toMap;
+
 public final class Table {
     public final String name;
 
     @Getter
     private final Collection<Row> rows;
 
-    public Table(String name, Collection<Row> rows) {
+    private Table(String name, Collection<Row> rows) {
         this.name = name;
-        this.rows = Collections.unmodifiableCollection(rows);
+        this.rows = unmodifiableCollection(rows);
+    }
+
+    public static Table ofName(String name) {
+        return new Table(name, emptyList());
+    }
+
+    public static Table of(String name, Row... rows) {
+        return new Table(name, Arrays.stream(rows).collect(toList()));
```

Comment:
There is shorter method - `java.util.Arrays#asList`

Reply:
:+1: